### PR TITLE
Refact ROBOT report page `robot_report.html`

### LINKED ISSUE #81: Dashboard says "See ROBOT Report for details." but those details are not in the report HTML.
On the OBI page of the dashboard <http://dashboard.obofoundry.org/dashboard/obi/dashboard.html> under "Textual Definitions" it says "3 missing definitions. See ROBOT Report for details." (This is a known problem with OBI that we are fixing <https://github.com/obi-ontology/obi/issues/1553>.) But when I click the link below for the ROBOT report <http://dashboard.obofoundry.org/dashboard/obi/robot_report.html>, the HTML version is truncated and I only see two warnings. The TSV version of the report is longer, but I don't see the textual definition there either.

I understand that the reports get long, but ideally at least these 3 missing definitions would be included in the report.

I think it would be better to have a "View Report" link in the "Textual Definition" row if there are any problems. The easy thing would be to make it the same link to the ROBOT report. It would be even nicer to have a view that's just for Textual Definition problems.

### LINKED ISSUE #101: dashboard error levels (level Barry Smith) in omo robot_report
I thinktheir might be some error in the Markdown converter as i spotted a really interesting dachboard report, where the error level was Barry Smith (not that i would not like to here that Barry Smith might be a category for semantic errors, but i think this was not intended).

A picture is shown below as well as the link to the report.

grafik

http://dashboard.obofoundry.org/dashboard/omo/robot_report.html

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #81 
Fixes #101

### CODE REVIEW

**Comment by matentzn** [comment]
@jsstevenson can you give this a eyeball review, you'd be better than me at this.

File: util/create_report_html.py [hunk 1]
```diff
@@ -110,9 +101,8 @@ def maybe_get_link(cell, context):
 }
 
 other_prefixes = {
-    'dc': 'http://purl.org/dc/terms/',
     'dcterms': 'http://purl.org/dc/terms/',
-    'dc11': 'http://purl.org/dc/elements/1.1/',
+    'dc': 'http://purl.org/dc/elements/1.1/',
```
Comment (matentzn) [inline util/create_report_html.py:105]: Why this change?

File: util/create_report_html.py [hunk 2]
```diff
@@ -93,11 +84,11 @@ def maybe_get_link(cell, context):
                 namespace = other_prefixes[prefix]
                 url = namespace + local_id
         # IRIs might be in angle brackets
-        iri = re.search(r'((http|https|ftp)://[^ <>]+)', cell)
+        iri = re.search(r'(http://purl.obolibrary.org/obo/[^ <>]+)', cell)
```
Comment (matentzn) [inline util/create_report_html.py:87]: What if the test is custom?

File: util/create_report_html.py
Comment (matentzn) [inline util/create_report_html.py:1]: Stupid question:

How redundant is this code with robot report ... -f html -o report.html?

**Review by jsstevenson — APPROVED** [review]
👍 looks good to me!

File: util/create_report_html.py
Comment (anitacaron) [reply util/create_report_html.py:1]: This code adds the type of errors and error breakdown, but the HTML output is the same.

File: util/create_report_html.py (hunk 2 above)
Comment (anitacaron) [reply util/create_report_html.py:87]: Ok, but if there's a URL inside a text, the text will be a link. I've seen this bug.

File: util/create_report_html.py (hunk 1 above)
Comment (anitacaron) [reply util/create_report_html.py:105]: renaming to the right name

File: util/create_report_html.py
Comment (anitacaron) [reply util/create_report_html.py:1]: but also it's limiting the number of rows

Comment (matentzn) [reply util/create_report_html.py:1]: Thanks for info.

File: util/create_report_html.py (hunk 2 above)
Comment (matentzn) [reply util/create_report_html.py:87]: I trust this change is ok, cant really understand the consequences.

File: util/create_report_html.py (hunk 1 above)
Comment (matentzn) [reply util/create_report_html.py:105]: "right" how? `dce` is elements, `dcterms` is terms. `dc` sounds a bit ambiguous to me.. Anyways, you can make that call without my approval!

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
cc @rays22 for next TWG report

File: util/create_report_html.py (hunk 1 above)
Comment (anitacaron) [reply util/create_report_html.py:105]: Actually, I've never seen xml/rdf file using dce, always dc.

Comment (anitacaron) [reply util/create_report_html.py:105]: I'll change dcterms to terms.
